modules/gapi/include/opencv2/gapi/fluid/gfluidbuffer.hpp

```diff
@@ -80,6 +78,7 @@ class GAPI_EXPORTS View
     Priv& priv();               // internal use only
     const Priv& priv() const;   // internal use only
 
+    View();
```

Comment:
How did this affect the python binding? I thought we don't wrap it now, right?

Reply:
These changes should fix compilation errors on window we faced when compiled g-api with python module